app/views/layouts/_header.html.erb

```diff
@@ -1,8 +1,11 @@
 <%= render './layouts/headness.html.haml' %>
 <% if user_signed_in? && current_user.admin %>
   <% if api_status == 'failed' %>
-    <div class="grid w-full bg-yellow-550 text-sm place-content-center py-1">
-      The Classrooms Update Script failed to run today. Please report the issue.
+    <div class="flex flex-row justify-center items-center bg-red-800 text-red-100 text-sm place-content-center py-1">
+      The Classrooms Update Script failed to run today.
+      <div>
+        <a id="myCustomTrigger" class="text-sm hover:underline px-1" href="#" data-email="<%= session[:user_email]%>">Report an Issue</a>
```

Comment:
I think that "**Report an Issue**" text should always be underlined and not just on hover. May be a dotted line by default which turns to solid line on hover. This way the user knows that the text is a link and is clickable.

Reply:
Solid underline always worked make this like the footer. Let's go that route